src/misc/misc.controller.ts

```diff
@@ -0,0 +1,45 @@
+import { DateService, FileService, XmlService } from './services';
+import type { Request } from 'express';
```

Comment:
Do you really need to explicitly install express just for this single typed import? You can avoid it like this:

```
RawBodyRequest<IncomingMessage>
```

Reply:
- `express` is also in use in `main.ts` for raw body parsing middleware
- `RawBodyRequest<IncomingMessage>` doesn't have the `body` inself

Reply:
* You don't need express; simply use the `rawBody` option and `useBodyParser` to configure what you need
* `RawBodyRequest` is designed especially to expose `rawBody` instead of `body`

Reply:
got you, thanks, adjusted